apps/storefront/docs/patterns/placement-and-order/placement-and-order.mdx

```diff
@@ -4,3 +4,27 @@ metaTitle: Placement and order
 mode: draft
 route: /patterns/placement-and-order/
 ---
+## Guidelines
+In order to place yout objects and component the right way, you have always to keep in mind the ready pattern of your user. 2 types of patterns can be encountered: F and Z.
+
+### The F reading pattern
+
+In the F-shaped scanning pattern is characterized by many fixations concentrated at the top and the left side of the page.
+
+Users first read in a horizontal movement, usually across the upper part of the content area. He will then
+move down the page a bit and then read across in a second horizontal movement that typically covers a shorter area than the previous movement. This additional element forms the F’s lower bar.
+
+The main interaction should then be aligned on the left.
+
+### The Z reading pattren
+
+The z-pattern layout follows the shape of the letter Z. A z-pattern design traces the route the human eye travels when they scan the page— left to right, top to bottom.
+
+Users first read in a horizontal movement, usually across the upper part of the content area then 
+down and to the left side of the page, creating a diagonal line and back across to the right again, forming a second horizontal line.
```

Comment:
Some changes to make it consistent with the previous paragraph: 

> The ~~z~~Z-shaped scanning pattern ~~layout follows the shape of the letter Z. A z-pattern design~~ traces the route the human eye travels when they scan the page — left to right, top to bottom.

Users first read in a horizontal movement, usually across the upper part of the content area. They will then move
down and to the left side of the page~~,~~ creating a diagonal line, and then move back across to the right again~~,~~ forming a second horizontal line.

Reply:
Updated